build_modules/lib/src/kernel_builder.dart

```diff
@@ -131,16 +136,19 @@ Future<void> _createKernel(
     @required String dartSdkDir,
     @required String sdkKernelPath,
     @required String librariesPath,
-    @required bool useIncrementalCompiler}) async {
+    @required bool useIncrementalCompiler,
+    @required bool trackUnusedInputs}) async {
   var request = WorkRequest();
   var scratchSpace = await buildStep.fetchResource(scratchSpaceResource);
   var outputId = module.primarySource.changeExtension(outputExtension);
   var outputFile = scratchSpace.fileFor(outputId);
 
   File packagesFile;
+  File usedInputsFile;
+  Map<String, AssetId> kernelInputPathToId;
```

Comment:
can we add a doc comment?

Reply:
I added some doc comments to these both and moved them into their own block (same for the ddc builder)